src/plugin.js

```diff
@@ -21,18 +21,14 @@
  *****************************************************************************/
 
 import YamcsHistoricalTelemetryProvider from './providers/historical-telemetry-provider.js';
-import RealtimeTelemetryProvider from './providers/realtime-telemetry-provider.js';
-import RealtimeEventsProvider from './providers/realtime-events-provider.js';
+import RealtimeProvider from './providers/realtime-provider.js';
 import YamcsObjectProvider from './providers/object-provider.js';
 import LimitProvider from './providers/limit-provider';
 
 
-import {
-    EVENTS_OBJECT_TYPE,
-    TELEMETRY_OBJECT_TYPE,
-    IMAGE_OBJECT_TYPE,
-    STRING_OBJECT_TYPE
-} from './const.js';
+import { OBJECT_TYPES } from './const';
+
+const REALTIME_TYPES = [OBJECT_TYPES.EVENTS_OBJECT_TYPE, OBJECT_TYPES.TELEMETRY_OBJECT_TYPE, OBJECT_TYPES.IMAGE_OBJECT_TYPE, OBJECT_TYPES.STRING_OBJECT_TYPE];
```

Comment:
`Object.values(OBJECT_TYPES)`?

Reply:
Thoughts on just moving this into the provider?

Reply:
Move it.

Reply:
Agreed, I'll move it. And utilize .values() if it makes sense once moved.